assets/src/components/Password.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	TextControl,
```

Comment:
This is a new Component, do we really gotta use something we _know_ we need to replace?